src/utils/update-manager.ts

```diff
@@ -392,6 +531,7 @@ export class UpdateManager {
         const newTabId: string | undefined = stateAndParams.state?.tabId;
         const isTabSwitched = isItemWithTabs(initiatorItem) && Boolean(newTabId);
         const currentMeta = getItemsStateAndParamsMeta(itemsStateAndParams);
+        const groupItemId = options?.groupItemId;
```

Comment:
What if we add `options.groupItemsIds` instead of `options.groupItemId`?
Maybe sometimes you don't need update all groups in queue or you want to control order in queue for update all groups.